lib/Client.js

```diff
@@ -286,6 +287,7 @@ class Client extends EventEmitter {
     */
     addMessageReaction(channelID, messageID, reaction, userID) {
         if(userID !== undefined) {
+            emitDeprecation("eris:1", "addMessageReaction() was called without an \"@me\" `userID` argument");
```

Comment:
I've made up our warning codes completely arbitrarily, just numbering them sequentially as I went. We could use more descriptive codes (e.g. associating them with issues/PRs that implement or describe the deprecation) or we could use a method like this; whatever works, as long as it's a standard naming scheme across the project.

Reply:
I feel like to make it harder to overlap in different prs we should use the pr/issue or an actual name or something

Reply:
Could probably discuss this in the discussion

Reply:
Discussed in https://github.com/abalabahaha/eris/discussions/1274#discussioncomment-1293529 and done in 672183b